tests/modules/asset_groups/resources/test_commit_asset_group.py

```diff
@@ -80,8 +80,12 @@ def test_commit_owner_asset_group(
         sighting = Sighting.query.get(sighting_uuid)
         encounters = sighting.get_encounters()
         assert len(encounters) == 2
-        assert encounters[0].owner == regular_user
-        assert encounters[1].owner == researcher_1
+        # It seems encounters may not be returned in order so we can't assert
```

Comment:
aside: so is this seemingly-arbitrary ordering here related to the problem @hwindsor was seeing in PR228?  i wonder if ordering is being returned based on guid or something (as that would be effectively random).

Reply:
Yeah, it's related.  I think we'd have to add a new field for ordering.